ta/remoteproc/include/user_ta_header_defines.h

```diff
@@ -0,0 +1,29 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) 2023 STMicroelectronics - All Rights Reserved
+ */
+
+#ifndef USER_TA_HEADER_DEFINES_H
+#define USER_TA_HEADER_DEFINES_H
+
+#include <ta_remoteproc.h>
+
+#define TA_UUID				TA_REMOTEPROC_UUID
+
+#define TA_FLAGS			(TA_FLAG_DEVICE_ENUM | \
+					 TA_FLAG_SINGLE_INSTANCE | \
+					 TA_FLAG_INSTANCE_KEEP_ALIVE)
+
+/* Provisioned stack size */
+#define TA_STACK_SIZE			(4 * 1024)
+
+/* Provisioned heap size for TEE_Malloc() and friends */
+#define TA_DATA_SIZE			(4 * 1024)
```

Comment:
4KByte is not much, is it enough for the TA?

Reply:
I have never seen any overflow. 
on the other side  it depends on the firmware/ platform  (for instance `TEE_MALLOC `to store the  header and the trailer of the image).
It might make more sense to define a config (as done for `CFG_PKCS11_TA_HEAP_SIZE`), no ?

Reply:
Yes, you can add such a config switch.

Reply:
Done